Sources/MessagingInApp/Gist/Managers/InlineMessageManager.swift

```diff
@@ -41,6 +41,8 @@ public class InlineMessageManager: BaseMessageManager {
         return gistView
     }
 
+    func stopAndCleanup() {}
```

Comment:
@Shahroz16 Added this empty placeholder to allow future implementation in manager as needed. Let me know if there's an alternative method that could be used here instead.

Reply:
lets add todo on it pls